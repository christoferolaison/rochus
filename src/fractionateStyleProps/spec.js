import fractionateStyleProps from '.'

test('fractionateStyleProps should return an empty props object and a sx object containing display key', () => {
  expect(fractionateStyleProps({ display: 'flex' })).toMatchSnapshot()
})

test('fractionateStyleProps should return an props object containing onClick and a sx object containing alignItems key', () => {
  expect(fractionateStyleProps({ alignItems: 'center', onClick: () => {} })).toMatchSnapshot()
})
