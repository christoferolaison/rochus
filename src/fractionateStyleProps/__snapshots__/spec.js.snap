// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fractionateStyleProps should return an empty props object and a sx object containing display key 1`] = `
Object {
  "props": Object {},
  "sx": Object {
    "alignContent": undefined,
    "alignItems": undefined,
    "alignSelf": undefined,
    "alignmentBaseline": undefined,
    "all": undefined,
    "animation": undefined,
    "animationDelay": undefined,
    "animationDirection": undefined,
    "animationDuration": undefined,
    "animationFillMode": undefined,
    "animationIterationCount": undefined,
    "animationName": undefined,
    "animationPlayState": undefined,
    "animationTimingFunction": undefined,
    "backfaceVisibility": undefined,
    "background": undefined,
    "backgroundAttachment": undefined,
    "backgroundBlendMode": undefined,
    "backgroundClip": undefined,
    "backgroundColor": undefined,
    "backgroundImage": undefined,
    "backgroundOrigin": undefined,
    "backgroundPosition": undefined,
    "backgroundPositionX": undefined,
    "backgroundPositionY": undefined,
    "backgroundRepeat": undefined,
    "backgroundRepeatX": undefined,
    "backgroundRepeatY": undefined,
    "backgroundSize": undefined,
    "baselineShift": undefined,
    "blockSize": undefined,
    "border": undefined,
    "borderBottom": undefined,
    "borderBottomColor": undefined,
    "borderBottomLeftRadius": undefined,
    "borderBottomRightRadius": undefined,
    "borderBottomStyle": undefined,
    "borderBottomWidth": undefined,
    "borderCollapse": undefined,
    "borderColor": undefined,
    "borderImage": undefined,
    "borderImageOutset": undefined,
    "borderImageRepeat": undefined,
    "borderImageSlice": undefined,
    "borderImageSource": undefined,
    "borderImageWidth": undefined,
    "borderLeft": undefined,
    "borderLeftColor": undefined,
    "borderLeftStyle": undefined,
    "borderLeftWidth": undefined,
    "borderRadius": undefined,
    "borderRight": undefined,
    "borderRightColor": undefined,
    "borderRightStyle": undefined,
    "borderRightWidth": undefined,
    "borderSpacing": undefined,
    "borderStyle": undefined,
    "borderTop": undefined,
    "borderTopColor": undefined,
    "borderTopLeftRadius": undefined,
    "borderTopRightRadius": undefined,
    "borderTopStyle": undefined,
    "borderTopWidth": undefined,
    "borderWidth": undefined,
    "bottom": undefined,
    "boxShadow": undefined,
    "boxSizing": undefined,
    "breakAfter": undefined,
    "breakBefore": undefined,
    "breakInside": undefined,
    "bufferedRendering": undefined,
    "captionSide": undefined,
    "caretColor": undefined,
    "clear": undefined,
    "clip": undefined,
    "clipPath": undefined,
    "clipRule": undefined,
    "color": undefined,
    "colorInterpolation": undefined,
    "colorInterpolationFilters": undefined,
    "colorRendering": undefined,
    "columnCount": undefined,
    "columnFill": undefined,
    "columnGap": undefined,
    "columnRule": undefined,
    "columnRuleColor": undefined,
    "columnRuleStyle": undefined,
    "columnRuleWidth": undefined,
    "columnSpan": undefined,
    "columnWidth": undefined,
    "columns": undefined,
    "contain": undefined,
    "content": undefined,
    "counterIncrement": undefined,
    "counterReset": undefined,
    "cursor": undefined,
    "cx": undefined,
    "cy": undefined,
    "d": undefined,
    "direction": undefined,
    "display": "flex",
    "dominantBaseline": undefined,
    "emptyCells": undefined,
    "fill": undefined,
    "fillOpacity": undefined,
    "fillRule": undefined,
    "filter": undefined,
    "flex": undefined,
    "flexBasis": undefined,
    "flexDirection": undefined,
    "flexFlow": undefined,
    "flexGrow": undefined,
    "flexShrink": undefined,
    "flexWrap": undefined,
    "float": undefined,
    "floodColor": undefined,
    "floodOpacity": undefined,
    "font": undefined,
    "fontDisplay": undefined,
    "fontFamily": undefined,
    "fontFeatureSettings": undefined,
    "fontKerning": undefined,
    "fontSize": undefined,
    "fontStretch": undefined,
    "fontStyle": undefined,
    "fontVariant": undefined,
    "fontVariantCaps": undefined,
    "fontVariantLigatures": undefined,
    "fontVariantNumeric": undefined,
    "fontWeight": undefined,
    "grid": undefined,
    "gridArea": undefined,
    "gridAutoColumns": undefined,
    "gridAutoFlow": undefined,
    "gridAutoRows": undefined,
    "gridColumn": undefined,
    "gridColumnEnd": undefined,
    "gridColumnGap": undefined,
    "gridColumnStart": undefined,
    "gridGap": undefined,
    "gridRow": undefined,
    "gridRowEnd": undefined,
    "gridRowGap": undefined,
    "gridRowStart": undefined,
    "gridTemplate": undefined,
    "gridTemplateAreas": undefined,
    "gridTemplateColumns": undefined,
    "gridTemplateRows": undefined,
    "height": undefined,
    "hyphens": undefined,
    "imageRendering": undefined,
    "inlineSize": undefined,
    "isolation": undefined,
    "justifyContent": undefined,
    "justifyItems": undefined,
    "justifySelf": undefined,
    "left": undefined,
    "letterSpacing": undefined,
    "lightingColor": undefined,
    "lineHeight": undefined,
    "listStyle": undefined,
    "listStyleImage": undefined,
    "listStylePosition": undefined,
    "listStyleType": undefined,
    "margin": undefined,
    "marginBottom": undefined,
    "marginLeft": undefined,
    "marginRight": undefined,
    "marginTop": undefined,
    "marker": undefined,
    "markerEnd": undefined,
    "markerMid": undefined,
    "markerStart": undefined,
    "mask": undefined,
    "maskType": undefined,
    "maxBlockSize": undefined,
    "maxHeight": undefined,
    "maxInlineSize": undefined,
    "maxWidth": undefined,
    "maxZoom": undefined,
    "minBlockSize": undefined,
    "minHeight": undefined,
    "minInlineSize": undefined,
    "minWidth": undefined,
    "minZoom": undefined,
    "mixBlendMode": undefined,
    "objectFit": undefined,
    "objectPosition": undefined,
    "offset": undefined,
    "offsetDistance": undefined,
    "offsetPath": undefined,
    "offsetRotate": undefined,
    "opacity": undefined,
    "order": undefined,
    "orientation": undefined,
    "orphans": undefined,
    "outline": undefined,
    "outlineColor": undefined,
    "outlineOffset": undefined,
    "outlineStyle": undefined,
    "outlineWidth": undefined,
    "overflow": undefined,
    "overflowAnchor": undefined,
    "overflowWrap": undefined,
    "overflowX": undefined,
    "overflowY": undefined,
    "padding": undefined,
    "paddingBottom": undefined,
    "paddingLeft": undefined,
    "paddingRight": undefined,
    "paddingTop": undefined,
    "page": undefined,
    "pageBreakAfter": undefined,
    "pageBreakBefore": undefined,
    "pageBreakInside": undefined,
    "paintOrder": undefined,
    "perspective": undefined,
    "perspectiveOrigin": undefined,
    "placeContent": undefined,
    "placeItems": undefined,
    "placeSelf": undefined,
    "pointerEvents": undefined,
    "position": undefined,
    "quotes": undefined,
    "r": undefined,
    "resize": undefined,
    "right": undefined,
    "rx": undefined,
    "ry": undefined,
    "shapeImageThreshold": undefined,
    "shapeMargin": undefined,
    "shapeOutside": undefined,
    "shapeRendering": undefined,
    "size": undefined,
    "speak": undefined,
    "src": undefined,
    "stopColor": undefined,
    "stopOpacity": undefined,
    "stroke": undefined,
    "strokeDasharray": undefined,
    "strokeDashoffset": undefined,
    "strokeLinecap": undefined,
    "strokeLinejoin": undefined,
    "strokeMiterlimit": undefined,
    "strokeOpacity": undefined,
    "strokeWidth": undefined,
    "tabSize": undefined,
    "tableLayout": undefined,
    "textAlign": undefined,
    "textAlignLast": undefined,
    "textAnchor": undefined,
    "textCombineUpright": undefined,
    "textDecoration": undefined,
    "textDecorationColor": undefined,
    "textDecorationLine": undefined,
    "textDecorationSkip": undefined,
    "textDecorationStyle": undefined,
    "textIndent": undefined,
    "textOrientation": undefined,
    "textOverflow": undefined,
    "textRendering": undefined,
    "textShadow": undefined,
    "textSizeAdjust": undefined,
    "textTransform": undefined,
    "textUnderlinePosition": undefined,
    "top": undefined,
    "touchAction": undefined,
    "transform": undefined,
    "transformOrigin": undefined,
    "transformStyle": undefined,
    "transition": undefined,
    "transitionDelay": undefined,
    "transitionDuration": undefined,
    "transitionProperty": undefined,
    "transitionTimingFunction": undefined,
    "unicodeBidi": undefined,
    "unicodeRange": undefined,
    "userSelect": undefined,
    "userZoom": undefined,
    "vectorEffect": undefined,
    "verticalAlign": undefined,
    "visibility": undefined,
    "webkitAppRegion": undefined,
    "webkitAppearance": undefined,
    "webkitBackgroundClip": undefined,
    "webkitBackgroundOrigin": undefined,
    "webkitBorderAfter": undefined,
    "webkitBorderAfterColor": undefined,
    "webkitBorderAfterStyle": undefined,
    "webkitBorderAfterWidth": undefined,
    "webkitBorderBefore": undefined,
    "webkitBorderBeforeColor": undefined,
    "webkitBorderBeforeStyle": undefined,
    "webkitBorderBeforeWidth": undefined,
    "webkitBorderEnd": undefined,
    "webkitBorderEndColor": undefined,
    "webkitBorderEndStyle": undefined,
    "webkitBorderEndWidth": undefined,
    "webkitBorderHorizontalSpacing": undefined,
    "webkitBorderImage": undefined,
    "webkitBorderStart": undefined,
    "webkitBorderStartColor": undefined,
    "webkitBorderStartStyle": undefined,
    "webkitBorderStartWidth": undefined,
    "webkitBorderVerticalSpacing": undefined,
    "webkitBoxAlign": undefined,
    "webkitBoxDecorationBreak": undefined,
    "webkitBoxDirection": undefined,
    "webkitBoxFlex": undefined,
    "webkitBoxFlexGroup": undefined,
    "webkitBoxLines": undefined,
    "webkitBoxOrdinalGroup": undefined,
    "webkitBoxOrient": undefined,
    "webkitBoxPack": undefined,
    "webkitBoxReflect": undefined,
    "webkitColumnBreakAfter": undefined,
    "webkitColumnBreakBefore": undefined,
    "webkitColumnBreakInside": undefined,
    "webkitFontSizeDelta": undefined,
    "webkitFontSmoothing": undefined,
    "webkitHighlight": undefined,
    "webkitHyphenateCharacter": undefined,
    "webkitLineBreak": undefined,
    "webkitLineClamp": undefined,
    "webkitLocale": undefined,
    "webkitLogicalHeight": undefined,
    "webkitLogicalWidth": undefined,
    "webkitMarginAfter": undefined,
    "webkitMarginAfterCollapse": undefined,
    "webkitMarginBefore": undefined,
    "webkitMarginBeforeCollapse": undefined,
    "webkitMarginBottomCollapse": undefined,
    "webkitMarginCollapse": undefined,
    "webkitMarginEnd": undefined,
    "webkitMarginStart": undefined,
    "webkitMarginTopCollapse": undefined,
    "webkitMask": undefined,
    "webkitMaskBoxImage": undefined,
    "webkitMaskBoxImageOutset": undefined,
    "webkitMaskBoxImageRepeat": undefined,
    "webkitMaskBoxImageSlice": undefined,
    "webkitMaskBoxImageSource": undefined,
    "webkitMaskBoxImageWidth": undefined,
    "webkitMaskClip": undefined,
    "webkitMaskComposite": undefined,
    "webkitMaskImage": undefined,
    "webkitMaskOrigin": undefined,
    "webkitMaskPosition": undefined,
    "webkitMaskPositionX": undefined,
    "webkitMaskPositionY": undefined,
    "webkitMaskRepeat": undefined,
    "webkitMaskRepeatX": undefined,
    "webkitMaskRepeatY": undefined,
    "webkitMaskSize": undefined,
    "webkitMaxLogicalHeight": undefined,
    "webkitMaxLogicalWidth": undefined,
    "webkitMinLogicalHeight": undefined,
    "webkitMinLogicalWidth": undefined,
    "webkitPaddingAfter": undefined,
    "webkitPaddingBefore": undefined,
    "webkitPaddingEnd": undefined,
    "webkitPaddingStart": undefined,
    "webkitPerspectiveOriginX": undefined,
    "webkitPerspectiveOriginY": undefined,
    "webkitPrintColorAdjust": undefined,
    "webkitRtlOrdering": undefined,
    "webkitRubyPosition": undefined,
    "webkitTapHighlightColor": undefined,
    "webkitTextCombine": undefined,
    "webkitTextDecorationsInEffect": undefined,
    "webkitTextEmphasis": undefined,
    "webkitTextEmphasisColor": undefined,
    "webkitTextEmphasisPosition": undefined,
    "webkitTextEmphasisStyle": undefined,
    "webkitTextFillColor": undefined,
    "webkitTextOrientation": undefined,
    "webkitTextSecurity": undefined,
    "webkitTextStroke": undefined,
    "webkitTextStrokeColor": undefined,
    "webkitTextStrokeWidth": undefined,
    "webkitTransformOriginX": undefined,
    "webkitTransformOriginY": undefined,
    "webkitTransformOriginZ": undefined,
    "webkitUserDrag": undefined,
    "webkitUserModify": undefined,
    "webkitWritingMode": undefined,
    "whiteSpace": undefined,
    "widows": undefined,
    "width": undefined,
    "willChange": undefined,
    "wordBreak": undefined,
    "wordSpacing": undefined,
    "wordWrap": undefined,
    "writingMode": undefined,
    "x": undefined,
    "y": undefined,
    "zIndex": undefined,
    "zoom": undefined,
  },
}
`;

exports[`fractionateStyleProps should return an props object containing onClick and a sx object containing alignItems key 1`] = `
Object {
  "props": Object {
    "onClick": [Function],
  },
  "sx": Object {
    "alignContent": undefined,
    "alignItems": "center",
    "alignSelf": undefined,
    "alignmentBaseline": undefined,
    "all": undefined,
    "animation": undefined,
    "animationDelay": undefined,
    "animationDirection": undefined,
    "animationDuration": undefined,
    "animationFillMode": undefined,
    "animationIterationCount": undefined,
    "animationName": undefined,
    "animationPlayState": undefined,
    "animationTimingFunction": undefined,
    "backfaceVisibility": undefined,
    "background": undefined,
    "backgroundAttachment": undefined,
    "backgroundBlendMode": undefined,
    "backgroundClip": undefined,
    "backgroundColor": undefined,
    "backgroundImage": undefined,
    "backgroundOrigin": undefined,
    "backgroundPosition": undefined,
    "backgroundPositionX": undefined,
    "backgroundPositionY": undefined,
    "backgroundRepeat": undefined,
    "backgroundRepeatX": undefined,
    "backgroundRepeatY": undefined,
    "backgroundSize": undefined,
    "baselineShift": undefined,
    "blockSize": undefined,
    "border": undefined,
    "borderBottom": undefined,
    "borderBottomColor": undefined,
    "borderBottomLeftRadius": undefined,
    "borderBottomRightRadius": undefined,
    "borderBottomStyle": undefined,
    "borderBottomWidth": undefined,
    "borderCollapse": undefined,
    "borderColor": undefined,
    "borderImage": undefined,
    "borderImageOutset": undefined,
    "borderImageRepeat": undefined,
    "borderImageSlice": undefined,
    "borderImageSource": undefined,
    "borderImageWidth": undefined,
    "borderLeft": undefined,
    "borderLeftColor": undefined,
    "borderLeftStyle": undefined,
    "borderLeftWidth": undefined,
    "borderRadius": undefined,
    "borderRight": undefined,
    "borderRightColor": undefined,
    "borderRightStyle": undefined,
    "borderRightWidth": undefined,
    "borderSpacing": undefined,
    "borderStyle": undefined,
    "borderTop": undefined,
    "borderTopColor": undefined,
    "borderTopLeftRadius": undefined,
    "borderTopRightRadius": undefined,
    "borderTopStyle": undefined,
    "borderTopWidth": undefined,
    "borderWidth": undefined,
    "bottom": undefined,
    "boxShadow": undefined,
    "boxSizing": undefined,
    "breakAfter": undefined,
    "breakBefore": undefined,
    "breakInside": undefined,
    "bufferedRendering": undefined,
    "captionSide": undefined,
    "caretColor": undefined,
    "clear": undefined,
    "clip": undefined,
    "clipPath": undefined,
    "clipRule": undefined,
    "color": undefined,
    "colorInterpolation": undefined,
    "colorInterpolationFilters": undefined,
    "colorRendering": undefined,
    "columnCount": undefined,
    "columnFill": undefined,
    "columnGap": undefined,
    "columnRule": undefined,
    "columnRuleColor": undefined,
    "columnRuleStyle": undefined,
    "columnRuleWidth": undefined,
    "columnSpan": undefined,
    "columnWidth": undefined,
    "columns": undefined,
    "contain": undefined,
    "content": undefined,
    "counterIncrement": undefined,
    "counterReset": undefined,
    "cursor": undefined,
    "cx": undefined,
    "cy": undefined,
    "d": undefined,
    "direction": undefined,
    "display": undefined,
    "dominantBaseline": undefined,
    "emptyCells": undefined,
    "fill": undefined,
    "fillOpacity": undefined,
    "fillRule": undefined,
    "filter": undefined,
    "flex": undefined,
    "flexBasis": undefined,
    "flexDirection": undefined,
    "flexFlow": undefined,
    "flexGrow": undefined,
    "flexShrink": undefined,
    "flexWrap": undefined,
    "float": undefined,
    "floodColor": undefined,
    "floodOpacity": undefined,
    "font": undefined,
    "fontDisplay": undefined,
    "fontFamily": undefined,
    "fontFeatureSettings": undefined,
    "fontKerning": undefined,
    "fontSize": undefined,
    "fontStretch": undefined,
    "fontStyle": undefined,
    "fontVariant": undefined,
    "fontVariantCaps": undefined,
    "fontVariantLigatures": undefined,
    "fontVariantNumeric": undefined,
    "fontWeight": undefined,
    "grid": undefined,
    "gridArea": undefined,
    "gridAutoColumns": undefined,
    "gridAutoFlow": undefined,
    "gridAutoRows": undefined,
    "gridColumn": undefined,
    "gridColumnEnd": undefined,
    "gridColumnGap": undefined,
    "gridColumnStart": undefined,
    "gridGap": undefined,
    "gridRow": undefined,
    "gridRowEnd": undefined,
    "gridRowGap": undefined,
    "gridRowStart": undefined,
    "gridTemplate": undefined,
    "gridTemplateAreas": undefined,
    "gridTemplateColumns": undefined,
    "gridTemplateRows": undefined,
    "height": undefined,
    "hyphens": undefined,
    "imageRendering": undefined,
    "inlineSize": undefined,
    "isolation": undefined,
    "justifyContent": undefined,
    "justifyItems": undefined,
    "justifySelf": undefined,
    "left": undefined,
    "letterSpacing": undefined,
    "lightingColor": undefined,
    "lineHeight": undefined,
    "listStyle": undefined,
    "listStyleImage": undefined,
    "listStylePosition": undefined,
    "listStyleType": undefined,
    "margin": undefined,
    "marginBottom": undefined,
    "marginLeft": undefined,
    "marginRight": undefined,
    "marginTop": undefined,
    "marker": undefined,
    "markerEnd": undefined,
    "markerMid": undefined,
    "markerStart": undefined,
    "mask": undefined,
    "maskType": undefined,
    "maxBlockSize": undefined,
    "maxHeight": undefined,
    "maxInlineSize": undefined,
    "maxWidth": undefined,
    "maxZoom": undefined,
    "minBlockSize": undefined,
    "minHeight": undefined,
    "minInlineSize": undefined,
    "minWidth": undefined,
    "minZoom": undefined,
    "mixBlendMode": undefined,
    "objectFit": undefined,
    "objectPosition": undefined,
    "offset": undefined,
    "offsetDistance": undefined,
    "offsetPath": undefined,
    "offsetRotate": undefined,
    "opacity": undefined,
    "order": undefined,
    "orientation": undefined,
    "orphans": undefined,
    "outline": undefined,
    "outlineColor": undefined,
    "outlineOffset": undefined,
    "outlineStyle": undefined,
    "outlineWidth": undefined,
    "overflow": undefined,
    "overflowAnchor": undefined,
    "overflowWrap": undefined,
    "overflowX": undefined,
    "overflowY": undefined,
    "padding": undefined,
    "paddingBottom": undefined,
    "paddingLeft": undefined,
    "paddingRight": undefined,
    "paddingTop": undefined,
    "page": undefined,
    "pageBreakAfter": undefined,
    "pageBreakBefore": undefined,
    "pageBreakInside": undefined,
    "paintOrder": undefined,
    "perspective": undefined,
    "perspectiveOrigin": undefined,
    "placeContent": undefined,
    "placeItems": undefined,
    "placeSelf": undefined,
    "pointerEvents": undefined,
    "position": undefined,
    "quotes": undefined,
    "r": undefined,
    "resize": undefined,
    "right": undefined,
    "rx": undefined,
    "ry": undefined,
    "shapeImageThreshold": undefined,
    "shapeMargin": undefined,
    "shapeOutside": undefined,
    "shapeRendering": undefined,
    "size": undefined,
    "speak": undefined,
    "src": undefined,
    "stopColor": undefined,
    "stopOpacity": undefined,
    "stroke": undefined,
    "strokeDasharray": undefined,
    "strokeDashoffset": undefined,
    "strokeLinecap": undefined,
    "strokeLinejoin": undefined,
    "strokeMiterlimit": undefined,
    "strokeOpacity": undefined,
    "strokeWidth": undefined,
    "tabSize": undefined,
    "tableLayout": undefined,
    "textAlign": undefined,
    "textAlignLast": undefined,
    "textAnchor": undefined,
    "textCombineUpright": undefined,
    "textDecoration": undefined,
    "textDecorationColor": undefined,
    "textDecorationLine": undefined,
    "textDecorationSkip": undefined,
    "textDecorationStyle": undefined,
    "textIndent": undefined,
    "textOrientation": undefined,
    "textOverflow": undefined,
    "textRendering": undefined,
    "textShadow": undefined,
    "textSizeAdjust": undefined,
    "textTransform": undefined,
    "textUnderlinePosition": undefined,
    "top": undefined,
    "touchAction": undefined,
    "transform": undefined,
    "transformOrigin": undefined,
    "transformStyle": undefined,
    "transition": undefined,
    "transitionDelay": undefined,
    "transitionDuration": undefined,
    "transitionProperty": undefined,
    "transitionTimingFunction": undefined,
    "unicodeBidi": undefined,
    "unicodeRange": undefined,
    "userSelect": undefined,
    "userZoom": undefined,
    "vectorEffect": undefined,
    "verticalAlign": undefined,
    "visibility": undefined,
    "webkitAppRegion": undefined,
    "webkitAppearance": undefined,
    "webkitBackgroundClip": undefined,
    "webkitBackgroundOrigin": undefined,
    "webkitBorderAfter": undefined,
    "webkitBorderAfterColor": undefined,
    "webkitBorderAfterStyle": undefined,
    "webkitBorderAfterWidth": undefined,
    "webkitBorderBefore": undefined,
    "webkitBorderBeforeColor": undefined,
    "webkitBorderBeforeStyle": undefined,
    "webkitBorderBeforeWidth": undefined,
    "webkitBorderEnd": undefined,
    "webkitBorderEndColor": undefined,
    "webkitBorderEndStyle": undefined,
    "webkitBorderEndWidth": undefined,
    "webkitBorderHorizontalSpacing": undefined,
    "webkitBorderImage": undefined,
    "webkitBorderStart": undefined,
    "webkitBorderStartColor": undefined,
    "webkitBorderStartStyle": undefined,
    "webkitBorderStartWidth": undefined,
    "webkitBorderVerticalSpacing": undefined,
    "webkitBoxAlign": undefined,
    "webkitBoxDecorationBreak": undefined,
    "webkitBoxDirection": undefined,
    "webkitBoxFlex": undefined,
    "webkitBoxFlexGroup": undefined,
    "webkitBoxLines": undefined,
    "webkitBoxOrdinalGroup": undefined,
    "webkitBoxOrient": undefined,
    "webkitBoxPack": undefined,
    "webkitBoxReflect": undefined,
    "webkitColumnBreakAfter": undefined,
    "webkitColumnBreakBefore": undefined,
    "webkitColumnBreakInside": undefined,
    "webkitFontSizeDelta": undefined,
    "webkitFontSmoothing": undefined,
    "webkitHighlight": undefined,
    "webkitHyphenateCharacter": undefined,
    "webkitLineBreak": undefined,
    "webkitLineClamp": undefined,
    "webkitLocale": undefined,
    "webkitLogicalHeight": undefined,
    "webkitLogicalWidth": undefined,
    "webkitMarginAfter": undefined,
    "webkitMarginAfterCollapse": undefined,
    "webkitMarginBefore": undefined,
    "webkitMarginBeforeCollapse": undefined,
    "webkitMarginBottomCollapse": undefined,
    "webkitMarginCollapse": undefined,
    "webkitMarginEnd": undefined,
    "webkitMarginStart": undefined,
    "webkitMarginTopCollapse": undefined,
    "webkitMask": undefined,
    "webkitMaskBoxImage": undefined,
    "webkitMaskBoxImageOutset": undefined,
    "webkitMaskBoxImageRepeat": undefined,
    "webkitMaskBoxImageSlice": undefined,
    "webkitMaskBoxImageSource": undefined,
    "webkitMaskBoxImageWidth": undefined,
    "webkitMaskClip": undefined,
    "webkitMaskComposite": undefined,
    "webkitMaskImage": undefined,
    "webkitMaskOrigin": undefined,
    "webkitMaskPosition": undefined,
    "webkitMaskPositionX": undefined,
    "webkitMaskPositionY": undefined,
    "webkitMaskRepeat": undefined,
    "webkitMaskRepeatX": undefined,
    "webkitMaskRepeatY": undefined,
    "webkitMaskSize": undefined,
    "webkitMaxLogicalHeight": undefined,
    "webkitMaxLogicalWidth": undefined,
    "webkitMinLogicalHeight": undefined,
    "webkitMinLogicalWidth": undefined,
    "webkitPaddingAfter": undefined,
    "webkitPaddingBefore": undefined,
    "webkitPaddingEnd": undefined,
    "webkitPaddingStart": undefined,
    "webkitPerspectiveOriginX": undefined,
    "webkitPerspectiveOriginY": undefined,
    "webkitPrintColorAdjust": undefined,
    "webkitRtlOrdering": undefined,
    "webkitRubyPosition": undefined,
    "webkitTapHighlightColor": undefined,
    "webkitTextCombine": undefined,
    "webkitTextDecorationsInEffect": undefined,
    "webkitTextEmphasis": undefined,
    "webkitTextEmphasisColor": undefined,
    "webkitTextEmphasisPosition": undefined,
    "webkitTextEmphasisStyle": undefined,
    "webkitTextFillColor": undefined,
    "webkitTextOrientation": undefined,
    "webkitTextSecurity": undefined,
    "webkitTextStroke": undefined,
    "webkitTextStrokeColor": undefined,
    "webkitTextStrokeWidth": undefined,
    "webkitTransformOriginX": undefined,
    "webkitTransformOriginY": undefined,
    "webkitTransformOriginZ": undefined,
    "webkitUserDrag": undefined,
    "webkitUserModify": undefined,
    "webkitWritingMode": undefined,
    "whiteSpace": undefined,
    "widows": undefined,
    "width": undefined,
    "willChange": undefined,
    "wordBreak": undefined,
    "wordSpacing": undefined,
    "wordWrap": undefined,
    "writingMode": undefined,
    "x": undefined,
    "y": undefined,
    "zIndex": undefined,
    "zoom": undefined,
  },
}
`;
